bin/needs-release.js

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env node
+/*
+ * (c) Copyright IBM Corp. 2021
+ * (c) Copyright Instana Inc. and contributors 2021
+ */
+
+'use strict';
```

Comment:
This file is used in the Github action to determine if a new release should be published. Otherwise we might publish releases that actually contain no real changes at all (only build changes etc.)